cmd/package-builder/package-builder.go

```diff
@@ -79,6 +79,11 @@ func runMake(args []string) error {
 			env.String("CONTROL_HOSTNAME", ""),
 			"the value that should be used when invoking the launcher's --control_hostname flag",
 		)
+		flDisableControlTLS = flagset.Bool(
+			"disable_control_tls",
+			env.Bool("DISABLE_CONTROL_TLS", false),
```

Comment:
should this be true for now at least? Lots of people building launcher packages, but the control server is kolide specific atm.

Reply:
Control is disabled by default I believe, so no need to have the TLS disabled by default for it

Reply:
ignore this comment.